# pdf-generator
[program:pdf-generator-ws]
user = root
autostart = true
autorestart = true
stdout_logfile = /var/log/pdf-generator/ws_out.log
stdout_logfile_backups = 3
stderr_logfile = /var/log/pdf-generator/ws_err.log
stderr_logfile_backups = 3
directory = /app
environment = PYTHON_EGG_CACHE=/python_eggs
command = /usr/bin/gunicorn manage:app --bind :8000 --workers 2

[program:pdf-generator-schedule]
user = root
autostart = true
autorestart = true
stdout_logfile = /var/log/pdf-generator/schedule_out.log
stdout_logfile_maxbytes = 2MB
stdout_logfile_backups = 3
stderr_logfile = /var/log/pdf-generator/schedule_err.log
stderr_logfile_backups = 3
directory = /app
environment = PYTHON_EGG_CACHE=/python_eggs
command = /usr/bin/python manage.py schedule

[program:pdf-generator-procesar-1]
user = root
autostart = true
autorestart = true
stdout_logfile = /var/log/pdf-generator/procesar-1_out.log
stdout_logfile_maxbytes = 2MB
stdout_logfile_backups = 3
stderr_logfile = /var/log/pdf-generator/procesar-1_err.log
directory = /app
environment = PYTHON_EGG_CACHE=/python_eggs,PDF_GENERATOR_WORKER_NAME=P1
command = /usr/bin/python manage.py procesar --proceso 1

[program:pdf-generator-procesar-2]
user = root
autostart = true
autorestart = true
stdout_logfile = /var/log/pdf-generator/procesar-2_out.log
stdout_logfile_maxbytes = 2MB
stdout_logfile_backups = 3
stderr_logfile = /var/log/pdf-generator/procesar-2_err.log
directory = /app
environment = PYTHON_EGG_CACHE=/python_eggs,PDF_GENERATOR_WORKER_NAME=P2
command = /usr/bin/python manage.py procesar --proceso 2

[program:pdf-generator-procesar-3]
user = root
autostart = true
autorestart = true
stdout_logfile = /var/log/pdf-generator/procesar-3_out.log
stdout_logfile_maxbytes = 2MB
stdout_logfile_backups = 3
stderr_logfile = /var/log/pdf-generator/procesar-3_err.log
directory = /app
environment = PYTHON_EGG_CACHE=/python_eggs,PDF_GENERATOR_WORKER_NAME=P3
command = /usr/bin/python manage.py procesar --proceso 3

[program:pdf-generator-procesar-4]
user = root
autostart = true
autorestart = true
stdout_logfile = /var/log/pdf-generator/procesar-4_out.log
stdout_logfile_maxbytes = 2MB
stdout_logfile_backups = 3
stderr_logfile = /var/log/pdf-generator/procesar-4_err.log
directory = /app
environment = PYTHON_EGG_CACHE=/python_eggs,PDF_GENERATOR_WORKER_NAME=P4
command = /usr/bin/python manage.py procesar --proceso 4

[program:mongodb]
user = root
autostart = true
autorestart = true
stdout_logfile = /var/log/pdf-generator/mongodb_out.log
stdout_logfile_maxbytes = 2MB
stdout_logfile_backups = 3
stderr_logfile = /var/log/pdf-generator/mongodb_err.log
stderr_logfile_backups = 3
stderr_logfile = /var/log/pdf-generator/mongodb_err.log
command = /usr/bin/mongod --config=/etc/mongod.conf
